assets/js/base/components/quantity-selector/style.scss

```diff
@@ -5,14 +5,14 @@
 	background: none transparent;
 
 	&:focus {
-		outline: 2px solid $core-grey-dark-100;
+		outline: 2px solid $core-grey-light-600;
```

Comment:
Yeah... It doesn't have enough contrast but since it's just a border for aesthetics I think it's ok in this case? I might be wrong, however.

I think it's similar to the borders between products in the _Cart_ block, which are also `$core-grey-light-600`: having them is for aesthetics but not something needed for functionality.